### PR TITLE
Profile stores

### LINKED ISSUE #13: Profile object to default User Store
Migrate profile from frontend to a default user store file

### PR DESCRIPTION
**Pull request by zain08816**
Resolves #13

Add the Profile object and new defaults into store.

#### WIP
- [x] fix rollup config to include store?
- [x] pass Profile as default store

### CODE REVIEW

**Comment by zain08816** [comment]
Currently, this branch temporarily moves Defaults to the Profile file. Rollup was misbehaving and recognizing the Defaults import into Profile a weird way in the cjs dist/index.js. Defaults *can* be left in Profile as many things that use Defaults also use Profile, but they should probably be separate stores in the end.

**Comment by zain08816** [comment]
> Currently, this branch temporarily moves Defaults to the Profile file. Rollup was misbehaving and recognizing the Defaults import into Profile a weird way in the cjs dist/index.js. Defaults _can_ be left in Profile as many things that use Defaults also use Profile, but they should probably be separate stores in the end.

I was sleeping and I realized what was wrong and how simple it was.

File: package/src/CoreProvider.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                );
+            } else {
+                return (
+                    <div key={currentId}>Error: Module {link.module} does not exist</div>
+                )
+            }
+        });
+    }
+    // populate the _store object with the default defined stores
+    _store = Store;
```
Comment (shivanmodha) [inline package/src/CoreProvider.js:33]: We want the user to be able to override the defaults.

**Review by shivanmodha — CHANGES_REQUESTED** [review]
Good job! Let's just fix the final store override features, and we good

**Review by shivanmodha — APPROVED** [review]

**Pull request merged**